### PR TITLE
Add set code samples for selected text and subject

### PR DESCRIPTION
**Pull request by TCourtneyOwen**
- Add set code samples for selected text and subject
- Combine getters and setters for selected text and subject and create get/set buttons

### CODE REVIEW

**Comment by ElizabethSamuel-MSFT** [comment]
@TCourtneyOwen You should consider deleting the entire Compose Basics section and moving subject samples to new folder based on organization in "Script lab snippets owners" OneNote page (Oleg emailed the link in one of his replies on the "Demo Outlook ScriptLab Integration" thread). If that organization doesn't work, let's discuss.

Thanks.

**Comment by TCourtneyOwen** [comment]
OK sounds good, I will just follow the guidance in the OneNote page

File: samples/outlook/90-other-item-apis/get-set-item-subject.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+id: outlook-other-item-apis-get-set-item-subject
+name: 'Gets, Sets Item Subject'
```
Comment (ElizabethSamuel-MSFT) [inline samples/outlook/90-other-item-apis/get-set-item-subject.yaml]:
```suggestion
name: 'Get and set the subject (Compose)'
```

File: samples/outlook/90-other-item-apis/get-set-item-subject.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+description: 'Gets, sets the subject of a compose item'
```
Comment (ElizabethSamuel-MSFT) [inline samples/outlook/90-other-item-apis/get-set-item-subject.yaml]:
```suggestion
description: 'Gets and sets the subject of an item in compose mode.'
```

File: samples/outlook/90-other-item-apis/get-set-item-subject.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                console.log(`Subject is ${result.value}`);
+            })
+        }
+
+        function set() {
+            let item = Office.context.mailbox.item as Office.ItemCompose;
+            let subject = 'Hello World!';
+            item.subject.setAsync(subject, (result: Office.AsyncResult) => {
+                if (result.status !== Office.AsyncResultStatus.Succeeded) {
+                console.error(`Action failed with message ${result.error.message}`);
```
Comment (ElizabethSamuel-MSFT) [inline samples/outlook/90-other-item-apis/get-set-item-subject.yaml]: nit: formatting - the statements in this if should be indented further

Comment (TCourtneyOwen) [reply samples/outlook/90-other-item-apis/get-set-item-subject.yaml]: fixed locally

File: samples/outlook/90-other-item-apis/get-set-item-subject.yaml (hunk 1 above)
Comment (TCourtneyOwen) [reply samples/outlook/90-other-item-apis/get-set-item-subject.yaml]: fixed locally

**Comment by ElizabethSamuel-MSFT** [comment]
@TCourtneyOwen I think you need to re-run `yarn start` and push those updates.

**Pull request merged**